utils/tsMorph/toObjectValue.ts

```diff
@@ -0,0 +1,93 @@
+import {
+    ArrayLiteralExpression,
+    NumericLiteral,
+    ObjectLiteralExpression,
+    PropertyAssignment,
+    SpreadAssignment,
+    StringLiteral,
+    SyntaxKind
+} from "ts-morph";
+
+/*
+ * Get js object value from ts-morph object literal expression
+ * */
+export const toObjectValue = (
```

Comment:
This looks like a lot of code, that took some time to put together. Questioning if this was really the most optimal way to do it.
Wouldn't the `getText` call on an object literal do the trick here? Then we just JSON.parse ?

Reply:
Very good observation! I will check this.

Reply:
I have tried this, at the first try didn't work, so in favor of time, I will leave this to experiment for the next upgrade.